servicebroker/test/util/apiServerUtil.test.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
```

Comment:
The file name should use '_' instead of '.' as that is the convention we are following across.